.github/workflows/auto-cherry-pick.yml

```diff
@@ -3,13 +3,13 @@ on:
   pull_request:
     types: [closed]
     branches:
-      - '4.1'
+      - '4.2'
 jobs:
   cherry-pick:
     strategy:
       fail-fast: false
       matrix:
-        target-branch: ['4.2', '4.3']
+        target-branch: ['4.1', '4.3']
```

Comment:
isn't there a risk of cascading cherry-picks?
Like 4.1 cherry-picks to 4.2, triggering another cherry-pick to 4.1 etc?